### PR TITLE
Team Teaching - Fix Avatar display names/billboards

### PR DESCRIPTION
**Pull request by Atlante45**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 31e565868d4657b1d97e5a0c4691416b17251df8 available at: https://gist.github.com/a8524ea390fec793a882<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5071.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5071.dmg

File: interface/src/avatar/Avatar.cpp [hunk 1]
```diff
@@ -604,44 +604,33 @@ void Avatar::renderBillboard() {
...
-    glm::vec3 axis = glm::axis(rotation);
-    glRotatef(glm::degrees(glm::angle(rotation)), axis.x, axis.y, axis.z);
-
+    
     // compute the size from the billboard camera parameters and scale
     float size = getBillboardSize();
-    glScalef(size, size, size);
+
+    gpu::Batch& batch = *renderArgs->_batch;
+    batch._glEnable(GL_TEXTURE_2D);
```
Comment (samcake) [inline interface/src/avatar/Avatar.cpp]: i think you don;t need that anymore

File: interface/src/avatar/Avatar.cpp [hunk 2]
```diff
@@ -604,44 +604,33 @@ void Avatar::renderBillboard() {
...
-
-    glPopMatrix();
-
-    glDisable(GL_TEXTURE_2D);
-    glEnable(GL_LIGHTING);
-    glDisable(GL_ALPHA_TEST);
-
-    glBindTexture(GL_TEXTURE_2D, 0);
+    batch._glBindTexture(GL_TEXTURE_2D, 0);
+    batch._glDisable(GL_TEXTURE_2D);
```
Comment (samcake) [inline interface/src/avatar/Avatar.cpp]: DOnlt do that

File: interface/src/avatar/Avatar.cpp [hunk 3]
```diff
@@ -604,44 +604,33 @@ void Avatar::renderBillboard() {
...
                                                         glm::vec4(1.0f, 1.0f, 1.0f, 1.0f));
-
-    glPopMatrix();
-
-    glDisable(GL_TEXTURE_2D);
-    glEnable(GL_LIGHTING);
-    glDisable(GL_ALPHA_TEST);
-
-    glBindTexture(GL_TEXTURE_2D, 0);
+    batch._glBindTexture(GL_TEXTURE_2D, 0);
```
Comment (samcake) [inline interface/src/avatar/Avatar.cpp]: don't do that

**Comment by hifi-gustavo** [comment]
Build console output for commit: 141d8fa0551cd97b4bc5f7e0a2952fedb90b921c available at: https://gist.github.com/a8524ea390fec793a882<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5071.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5071.dmg

**Comment by Atlante45** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 141d8fa0551cd97b4bc5f7e0a2952fedb90b921c available at: https://gist.github.com/a8524ea390fec793a882<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5071.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5071.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: c8c372bf3ae4363576739f1dec61623b446f74ed available at: https://gist.github.com/a8524ea390fec793a882<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5071.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5071.dmg

**Pull request merged**